azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/CallCompositeActivity.kt

```diff
@@ -75,6 +76,13 @@ internal class CallCompositeActivity : AppCompatActivity() {
                 configuration.themeConfig?.theme!!, true
             )
         }
+
+        configuration.localizationConfig?.let { localeConfig ->
+            Locale.setDefault(Locale(localeConfig.language))
+            window?.decorView?.layoutDirection =
```

Comment:
Can we check what window?.decorView?.layoutDirection is already applied. If that value is different, only then assign it